extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -51,63 +51,62 @@ class AchConfig {
 	private $mNotInTrackStatic;
 
 
-
 	public function __construct() {
 		//$this->mLoadedData = array();
 		$this->reset();
 
 		$this->mNotInTrackStatic = array(
-			BADGE_WELCOME => array('level' => BADGE_LEVEL_BRONZE, 'infinite' => false),
-			BADGE_INTRODUCTION => array('level' => BADGE_LEVEL_BRONZE, 'infinite' => false),
-			BADGE_SAYHI => array('level' => BADGE_LEVEL_BRONZE, 'infinite' => false),
-			BADGE_CREATOR => array('level' => BADGE_LEVEL_GOLD, 'infinite' => false),
-			BADGE_POUNCE => array('level' => BADGE_LEVEL_SILVER, 'infinite' => false),
-			BADGE_CAFFEINATED => array('level' => BADGE_LEVEL_SILVER, 'infinite' => false),
-			BADGE_LUCKYEDIT => array('level' => BADGE_LEVEL_GOLD, 'infinite' => true)
+			BADGE_WELCOME => array( 'level' => BADGE_LEVEL_BRONZE, 'infinite' => false ),
+			BADGE_INTRODUCTION => array( 'level' => BADGE_LEVEL_BRONZE, 'infinite' => false ),
+			BADGE_SAYHI => array( 'level' => BADGE_LEVEL_BRONZE, 'infinite' => false ),
+			BADGE_CREATOR => array( 'level' => BADGE_LEVEL_GOLD, 'infinite' => false ),
+			BADGE_POUNCE => array( 'level' => BADGE_LEVEL_SILVER, 'infinite' => false ),
+			BADGE_CAFFEINATED => array( 'level' => BADGE_LEVEL_SILVER, 'infinite' => false ),
+			BADGE_LUCKYEDIT => array( 'level' => BADGE_LEVEL_GOLD, 'infinite' => true )
 		);
 
 		$this->mInTrackStatic = array(
 			BADGE_EDIT => array(
 				'laps' => array(
-					array('level' => BADGE_LEVEL_BRONZE, 'events' => 1),
-					array('level' => BADGE_LEVEL_BRONZE, 'events' => 5),
-					array('level' => BADGE_LEVEL_BRONZE, 'events' => 10),
-					array('level' => BADGE_LEVEL_SILVER, 'events' => 25),
-					array('level' => BADGE_LEVEL_SILVER, 'events' => 50),
-					array('level' => BADGE_LEVEL_SILVER, 'events' => 100),
-					array('level' => BADGE_LEVEL_GOLD, 'events' => 250),
-					array('level' => BADGE_LEVEL_GOLD, 'events' => 500)
+					array( 'level' => BADGE_LEVEL_BRONZE, 'events' => 1 ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "events" 35 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)